src/payment/strategies/stripev3/stripev3-initialize-options.ts

```diff
@@ -12,7 +12,7 @@ export default interface StripeV3PaymentInitializeOptions {
     /**
      * The location to insert the credit card number form field.
      */
-    containerId: string;
+    containerId: ElementOrSelector;
```

Comment:
is this a breaking change?

it's a bit misleading that a `containerId` property can be an `element` or a `selector`

Even if it's just a selector, the selector would be `#inputId` while the containerId would simply be `inputId`

Reply:
> is this a breaking change?
> 
> it's a bit misleading that a `containerId` property can be an `element` or a `selector`
> 
> Even if it's just a selector, the selector would be `#inputId` while the containerId would simply be `inputId`
> 
> maybe we should deprecate `containerId` and create a new one `container: ElementOrSelector ` ? And explain properly in the description that it could either be an element or a selector (not an ID)

I don't believe this is a breaking change as we've always been using `mount` with a #